src/View/mainpage/MainPage.py

```diff
@@ -221,3 +221,19 @@ def zoom_out(self, is_four_view):
             self.dicom_view_sagittal.zoom_out()
         else:
             self.dicom_single_view.zoom_out()
+
+    def format_data(self, size):
+        self.dicom_view_axial.format_metadata(size)
+
+
+class StackedWidget(QStackedWidget):
```

Comment:
consider changing the class name to DicomViewQStackedWidget. Also, put this class to a separate file (src.mainpage.DicomViewQStackedWidget)

Reply:
Fixed. Thanks for your comment! 👍